src/nameserver/name_server_impl.cc

```diff
@@ -5445,7 +5445,7 @@ void NameServerImpl::OnLocked() {
     if (!Recover()) {
         PDLOG(WARNING, "recover failed");
     }
-    if (IsClusterMode() && (db_table_info_.empty() || db_table_info_[INTERNAL_DB].empty())) {
+    if (IsClusterMode() && (db_table_info_.empty() || db_table_info_[INTERNAL_DB].empty() || db_table_info_[INFORMATION_SCHEMA_DB].empty())) {
```

Comment:
what if INTERNAL_DB exist already and INFORMATION_SCHEMA is not, will this fatal ?